packages/xo-server-audit/src/index.js

```diff
@@ -238,6 +240,13 @@ class AuditXoPlugin {
     clean.permission = 'admin'
     clean.description = 'Clean audit database'
 
+    const removeRecord = concurrency(1)(this._removeRecord.bind(this))
```

Comment:
It works but you are not looking to limit the concurrency but to create a synchronized function :slightly_smiling_face: 

https://github.com/JsCommunity/decorator-synchronized